### PR TITLE
New post: Hugging Face and IBM

### PR DESCRIPTION
**Pull request by juliensimon**
I applied the feedback from IBM, this should be good to go.

### CODE REVIEW

File: huggingface-and-ibm.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Hugging Face and IBM partner on watsonx.ai, the next-generation enterprise studio for AI builders"
+thumbnail: /blog/assets/144_ibm/01.png
+authors:
+- user: juliensimon
```
Comment (julien-c) [inline huggingface-and-ibm.md:5]: any co-authors from the hf.co/ibm org?

File: assets/144_ibm/01.png
Comment (philschmid) [inline assets/144_ibm/01.png:1]: Thats an awesome image! 🤗

**Review by philschmid — APPROVED** [review]
✅ 
Any eta when this will be live?

File: assets/144_ibm/01.png
Comment (pcuenca) [inline assets/144_ibm/01.png:1]: Nice! But maybe we can reduce the file size using a tool such as [ImageOptim](https://imageoptim.com). Not sure if it'd still look good if we downscale it, I read in the [README](https://github.com/huggingface/blog#how-to-get-a-nice-responsive-thumbnail) that the recommended size for thumbnails is `1300x650`. I don't think it has to be a `png` either, maybe a `jpg` works best here.

File: huggingface-and-ibm.md (hunk 1 above)
Comment (pcuenca) [reply huggingface-and-ibm.md:5]: Reminder to add the post entry to `_blog.yml` in the root of the repo :)

File: huggingface-and-ibm.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+# Hugging Face and IBM partner on watsonx.ai, the next-generation enterprise studio for AI builders
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+<kbd>
+  <img src="assets/144_ibm/01.png">
+</kbd>
```
Comment (pcuenca) [inline huggingface-and-ibm.md:16]: Interesting! What does putting the image inside `kbd` achieve? (Just curious).

File: huggingface-and-ibm.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+All hype aside, it's hard to deny the profound impact that AI is having on society and businesses. From startups to enterprises to the public sector, every customer we talk to is busy experimenting with large language models and generative AI, identifying the most promising use cases, and gradually bringing them to production. 
+
+The #1 thing that customers tell us is that no single model will rule them all. They now understand the value of building the best model for each use case to maximize its relevance on company data while optimizing the compute budget. Of course, privacy and intellectual property are also top concerns, and customers want to ensure they maintain complete control.
```
Comment (pcuenca) [inline huggingface-and-ibm.md:20]:
```suggestion
The #1 comment we get from customers is that no single model will rule them all. They understand the value of building the best model for each use case to maximize its relevance on company data while optimizing the compute budget. Of course, privacy and intellectual property are also top concerns, and customers want to ensure they maintain complete control.
```

File: huggingface-and-ibm.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+3. **Open-source libraries**: the Hugging Face [libraries](https://huggingface.co/docs) let you download pre-trained models with a single line of code, and you can start experimenting with your data in minutes. From training to deployment to hardware optimization, customers can rely on a consistent set of community-driven tools that work the same everywhere, from their laptops to their production environment.
+
+In addition, our cloud partnerships let customers use Hugging Face models and libraries at any scale without worrying about provisioning infrastructure and building technical environments. This makes it much easier to get high-quality models out the door at a rapid pace without having to reinvent the wheel.
+
+Following up on our collaboration with AWS on Amazon SageMaker and Microsoft on Azure Machine Learning, we're thrilled to work with none other than IBM on their new AI studio, [watsonx.ai](https://www.ibm.com/products/watsonx-ai). [watsonx.ai](http://watsonx.ai) is the next-generation enterprise studio for AI builders to train, validate, tune, and deploy both traditional ML and new generative AI capabilities, powered by foundation models.
```
Comment (pcuenca) [inline huggingface-and-ibm.md:33]:
```suggestion
Following up on Hugging Face collaboration with AWS on Amazon SageMaker, and with Microsoft on Azure Machine Learning, we're thrilled to work with none other than IBM on their new AI studio, [watsonx.ai](https://www.ibm.com/products/watsonx-ai). [watsonx.ai](http://watsonx.ai) is the next-generation enterprise studio for AI builders to train, validate, tune, and deploy both traditional ML and new generative AI capabilities, powered by foundation models.
```

Just to remove any potential ambiguity, as the post looks like a joint announcement.

File: huggingface-and-ibm.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Under the hood, watsonx.ai also integrates many Hugging Face open-source libraries, such as [transformers](https://github.com/huggingface/transformers) (100k+ GitHub stars!), [accelerate](https://github.com/huggingface/accelerate), [peft](https://github.com/huggingface/peft) and our [Text Generation Inference](https://github.com/huggingface/text-generation-inference) server, to name a few. We're happy to partner with IBM and to collaborate on the watsonx AI and data platform so that Hugging Face customers can work natively with their Hugging Face models and datasets to multiply the impact of AI across businesses.
```
Comment (pcuenca) [inline huggingface-and-ibm.md:37]: A bit too long-winded maybe? How about something like

```suggestion
Watsonx.ai already incorporates many Hugging Face open-source libraries, including [transformers](https://github.com/huggingface/transformers) (100k+ GitHub stars!), [accelerate](https://github.com/huggingface/accelerate), [peft](https://github.com/huggingface/peft), and our optimized [Text Generation Inference](https://github.com/huggingface/text-generation-inference) server. By partnering with IBM and further collaborating on the watsonx AI and data platform, we enable Hugging Face customers to seamlessly leverage their existing models and datasets, amplifying the impact of AI across businesses.
```

File: huggingface-and-ibm.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+IBM also developed its slate of Large Language Models, and we will work with their team to open-source them and make them easily available in Hugging Face.
```
Comment (pcuenca) [inline huggingface-and-ibm.md:39]:
```suggestion
In addition, IBM has also developed its own collection of Large Language Models, and we will work with their team to open-source them and make them easily available in the Hugging Face Hub.
```

**Review by pcuenca — COMMENTED** [review]
Great work! Made a few minor suggestions :)

File: huggingface-and-ibm.md (hunk 1 above)
Comment (juliensimon) [reply huggingface-and-ibm.md:5]: Sure. I always do it at the last minute to avoid conflicts.

File: huggingface-and-ibm.md (hunk 2 above)
Comment (juliensimon) [reply huggingface-and-ibm.md:16]: It adds a border around the image (see https://github.com/juliensimon/blog/blob/ibm/huggingface-and-ibm.md). That doesn't appear on our blog, for some reason.

**Comment by juliensimon** [comment]
@philschmid I'll merge it tomorrow morning. The Azure thing is coming out today, right?

File: assets/144_ibm/01.png
Comment (juliensimon) [reply assets/144_ibm/01.png:1]: Resized

**Pull request merged**